pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StudySql.java

```diff
@@ -16,6 +16,14 @@ int insertSettings(
             @Bind("studyId") long studyId,
             @Bind("inviteErrorTmplId") Long inviteErrorTemplateId);
 
+    @SqlUpdate("insert into study_settings (umbrella_study_id, invite_error_template_id, analytics_enabled, analytics_token)"
+            + " values (:studyId, :inviteErrorTmplId, :analyticsEnabled, :analyticsToken)")
+    int insertSettings(
```

Comment:
Can you get rid of the `insertSettings()` method above and just have one of these?